docker-compose.yml

```diff
@@ -108,6 +108,8 @@ services:
       - PRODUCT_CATALOG_SERVICE_ADDR
       - SHIPPING_SERVICE_ADDR
       - OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+      - OTEL_EXPORTER_OTLP_METRICS_ENDPOINT
```

Comment:
is this related to this PR?

Reply:
This makes sure that the env var - if configured - is passed on to the `checkoutservice` container, where the Go SDK will automatically pick up the value to use the respective endpoint. Since this comes from the SDK OOTB, it does not involve any code changes in the implementation itself.
See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#configuration-options.

Reply:
Yes, this ENV is picked in the golang otel sdk